tests/TextDiffTest.php

```diff
@@ -36,7 +36,9 @@
      */
     public function testDiffReportingDecorated($expected, $format, $isDecorated)
     {
-        $command = new FixCommand();
+        $command = new FixCommand(
+            $this->prophesize('PhpCsFixer\ToolInfo')->reveal()
```

Comment:
does this test need a `prophesize` version of `ToolInfo`?

Reply:
Not sure, I'll have a look.

Reply:
Removed unneeded dummies.